packages/select/src/components/multi-select/multiSelect.tsx

```diff
@@ -195,19 +206,32 @@ export class MultiSelect<T> extends AbstractPureComponent<MultiSelectProps<T>, M
                 onQueryChange={this.handleQueryChange}
                 ref={this.refHandlers.queryList}
                 renderer={this.renderQueryList}
+                onKeyUp={customTarget != null ? this.keyUpHandler : undefined}
+                onKeyDown={customTarget != null ? this.keyDownHandler : undefined}
```

Comment:
I don't think this is the right place for these - currently they're on the element that wraps the target, even with a custom target I think we'd want to keep them there

I'm seeing keyboard nav isn't working on the docs now but I think it was before